docs/source/commandline/commandline-all.rst

```diff
@@ -1343,69 +1818,135 @@ Core Input Options
+    Enable user interface on top of emulated keyboard (if present).
+
+    The default is OFF (**-noui_active**)
 
 .. _mame-commandline-nooffscreenreload:
 
 **-[no]offscreen_reload** / **-[no]reload**
 
-	Controls whether or not MAME treats a second button input from a lightgun as a reload signal. In this case, MAME will report the gun's position as (0,MAX) with the trigger held, which is equivalent to an	offscreen reload. This is only needed for games that required you to shoot offscreen to reload, and then only if your gun does not support off screen reloads. The default is OFF (**-nooffscreen_reload**).
+    Controls whether or not MAME treats a second button input from a lightgun as
+    a reload signal.  In this case, MAME will report the gun's position as
+    (0,MAX) with the trigger held, which is equivalent to an offscreen reload.
+
+    This is only needed for games that required you to shoot offscreen to
+    reload, and then only if your gun does not support off screen reloads.
+
+    The default is OFF (**-nooffscreen_reload**).
 
 .. _mame-commandline-joystickmap:
 
 **-joystick_map** *<map>* / **-joymap** *<map>*
 
-	Controls how joystick values map to digital joystick controls. MAME accepts all joystick input from the system as analog data. For true analog joysticks, this needs to be mapped down to the usual 4-way or 8-way digital joystick values. To do this, MAME divides the analog range into a 9x9 grid. It then takes the joystick axis position (for X and Y axes only), maps it to this grid, and then looks up a translation from a joystick map. This parameter allows you to specify the map. The default is ``auto``, which means that a standard 8-way, 4-way, or 4-way diagonal map is selected automatically based on the input port configuration of the current system.
+    Controls how joystick values map to digital joystick controls. MAME accepts
+    all joystick input from the system as analog data.  For true analog
+    joysticks, this needs to be mapped down to the usual 4-way or 8-way digital
+    joystick values.  To do this, MAME divides the analog range into a 9x9 grid.
+    It then takes the joystick axis position (for X and Y axes only), maps it to
+    this grid, and then looks up a translation from a joystick map.  This
+    parameter allows you to specify the map.
 
-	Maps are defined as a string of numbers and characters. Since the grid is 9x9, there are a total of 81 characters necessary to define a	complete map. Below is an example map for an 8-way joystick:
+    The default is ``auto``, which means that a standard 8-way, 4-way, or 4-way
+    diagonal map is selected automatically based on the input port configuration
+    of the current system.
+
+    Maps are defined as a string of numbers and characters. Since the grid is
+    9x9, there are a total of 81 characters necessary to define a	complete map.
+    Below is an example map for an 8-way joystick:
```

Comment:
Is *<map>* supposed to be a filename, or are you supposed to jam the map into the command-line argument?  It isn't clear here.  If these maps are example file fragments, use `::` (double colon) at the end of this line and extra indentation so they show in a code sample box.

Reply:
This is stuff I've never used, so I clearly need to research this before cleaning it up.